app/src/main/res/layout/activity_main.xml

```diff
@@ -1,118 +1,178 @@
-        android:id="@+id/thumbnail_text"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/select_file"
-        android:layout_centerHorizontal="true"
-        android:layout_marginTop="20dp"
-        android:text="@string/thumbnail_of_youtube_video"
-        android:textAlignment="center"
-        android:textSize="20sp" />
-
-    <EditText
-        android:id="@+id/youtube_link"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/thumbnail_text"
-        android:hint="@string/youtube_link"
-        android:inputType="textUri"
-        android:textAlignment="center"
-        android:textSize="15sp" />
+        android:text="@string/bitmap_from_pdf"
+        android:textColor="@color/white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/textView" />
 
     <Button
-        android:id="@+id/generate_thumbnail_for_youtube"
+        android:id="@+id/youtube_link"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@id/youtube_link"
-        android:layout_centerInParent="true"
+        android:layout_marginTop="50dp"
         android:backgroundTint="@color/black"
-        android:text="@string/generate_thumbnail" />
-
-    <TextView
-        android:id="@+id/thumbnail_of_vdo"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_below="@id/generate_thumbnail_for_youtube"
-        android:layout_centerHorizontal="true"
-        android:layout_marginTop="20dp"
-        android:text="@string/thumbnail_of_video"
-        android:textSize="20sp" />
+        android:text="@string/bitmap_from_youtube"
+        android:textColor="@color/white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/pdf" />
 
     <Button
-        android:id="@+id/select_video"
+        android:id="@+id/video_uri"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_below="@id/thumbnail_of_vdo"
-        android:layout_centerHorizontal="true"
-        android:layout_centerVertical="true"
+        android:layout_marginTop="50dp"
         android:backgroundTint="@color/black"
-        android:text="@string/select_video" />
+        android:text="@string/bitmap_of_video"
+        android:textColor="@color/white"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/youtube_link" />
+
+</androidx.constraintlayout.widget.ConstraintLayout>
+
+
+
+<!--<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"-->
```

Comment:
Remove all this commented out code from the XML.

Reply:
done